### PR TITLE
NO-ISSUE: Add ocs operator unit test for standard deployment mode

### PR DESCRIPTION
**Pull request by rewantsoni**
Signed-off-by: Rewant Soni <resoni@redhat.com>

### CODE REVIEW

File: internal/operators/ocs/validation_test.go [hunk 1]
```diff
@@ -1051,6 +1051,39 @@ var _ = Describe("Ocs Operator use-cases", func() {
...
+				{ID: &hid1, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleMaster},
+				{ID: &hid2, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleMaster},
+				{ID: &hid3, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleMaster},
+				{ID: &hid4, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(10, 32000000000), Role: models.HostRoleWorker},
+				{ID: &hid5, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(10, 32000000000), Role: models.HostRoleWorker},
+				{ID: &hid6, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(10, 32000000000), Role: models.HostRoleWorker},
+			},
+			statusInfoChecker: makeValueChecker(clust.StatusInfoReady),
+			validationsChecker: makeJsonChecker(map[clust.ValidationID]validationCheckResult{
+				clust.IsMachineCidrDefined:                {status: clust.ValidationSuccess, messagePattern: "The Machine Network CIDR is defined"},
```
Comment (priyanka19-98) [inline internal/operators/ocs/validation_test.go:1073]: Please rename `clust` to readable name `cluster`.

Comment (rewantsoni) [reply internal/operators/ocs/validation_test.go:1073]: We have cluster defined https://github.com/openshift/assisted-service/blob/0076817559528d2d4c266ddc9dda5e7392ece26b/internal/operators/ocs/validation_test.go#L279 here, can you suggest a different name?

Comment (priyanka19-98) [reply internal/operators/ocs/validation_test.go:1073]: I could suggest `testCluster`. @nbalacha , any other suggestions?

Comment (nbalacha) [reply internal/operators/ocs/validation_test.go:1073]: I see all the other tests are using clust so maybe we can continue with that.

**Review by nbalacha — APPROVED** [review]
/lgtm

**Review by gamli75 — APPROVED** [review]

**Pull request merged**